app/models/target_remote_execution_proxy.rb

```diff
@@ -0,0 +1,4 @@
+class TargetRemoteExecutionProxy < ActiveRecord::Base
+  belongs_to :remote_execution_proxy, :class_name => 'SmartProxy'
+  belongs_to :target, :polymorphic => true
```

Comment:
Yeah, polymorphic to add host later, was going to add it in a separate PR, unless you really want it here